canonical-websites/docs.jujucharms.com
# Add commands page

**This PR is based on #8 - merge that first**

This add a static commands page, as it won't fit in Discourse.

Fixes https://github.com/canonical-websites/docs.jujucharms.com/issues/9

QA
--

`./run`, go to http://0.0.0.0:8033/commands - see it looks a lot like https://docs.jujucharms.com/devel/en/commands.

Go to http://0.0.0.0:8033/devel/en/commands - see you're redirected to /commands.
